client/views/jobs/job.html

```diff
@@ -51,7 +51,13 @@
 							<hr>
 							<i class="fa fa-trophy fa-fw"></i> Featured Until {{formatDate featuredThrough}}
 							{{/if}}
-						</div>
+              <hr>
+              {{#if notInterested }}
+                <a id="im-interested" class="btn btn-primary">I'm interested in this</a>
```

Comment:
it would be nice if this was icons instead of button/text

maybe a grey or hollow heart for default state with a solid red heart for 'interested in'

could put the text you have as tooltips on the icons